src/quo2/components/settings/reorder_item/view.cljs

```diff
@@ -8,7 +8,7 @@
 
 (defn reorder-item
   [item type {:keys [blur? drag]}]
-  (case type
+  (condp = type
```

Comment:
From the [linter issue](https://github.com/clj-kondo/clj-kondo/issues/2111), symbol case constants in CLJS are _"never evaluated except when the symbol refers to a constant"_, and here `types/...` are constants.

Reply:
@clauxx, take a look at this code:

```clojure
(let [x :option-1]
  (case x
    banana :is-banana
    apple :is-apple
    nil))
```

Both `banana` and `apple` are vars that don't exist. The Google Closure compiler won't fail, but the code will throw an exception in production. The linter won't catch errors either. So it's very very easy for developers to mistakenly refactor code and not update the conditions because they think the compiler will fail (as it does when using `cond`, `condp`, etc).

`case` is faster than `cond` or `condp`, but it should be used only with literals. It's the first thing mentioned in the Clojure's docstring. We had bugs in production before, so enabling this linter gets rid of all this.

You'll still be able to use `case`, but only with literal values.

Reply:
makes sense. Just tested it and wow, good to know about `case`.

Reply:
Cool! Thanks for reviewing @clauxx

Reply:
Thanks for the good explanation!